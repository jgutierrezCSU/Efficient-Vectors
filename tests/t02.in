findRange
5 10
5 2 7 4 3
findRange
10 94
72	34	41	85	97	61	55	27	3	60
findRange
10  83
15	34	41	85	60	61	55	27	72	97
