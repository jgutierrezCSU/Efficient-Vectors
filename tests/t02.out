5 2 7 4 3 
10 is greater than the range.
72 34 41 85 97 61 55 27 3 60 
94 is NOT greater than the range.
15 34 41 85 60 61 55 27 72 97 
83 is NOT greater than the range.
